package-lock.json

```diff
@@ -0,0 +1,1160 @@
+{
```

Comment:
You can delete this file now that there's `yarn.lock` in the repo.

Reply:
Also, looks like you need to remove `modifiedRuleIds.txt` file as well.

Reply:
@akonchady Done, removed and merged with `master`.